q2_diversity/_alpha/alpha_correlation_assets/src/render.js

```diff
@@ -17,12 +17,19 @@ export function render(svg, data) {
 
   const { xAxisLabel, yAxisLabel, minX, maxX, minY, maxY } = data;
 
-  const x = scaleLinear().domain([minX - ((maxX - minX) * 0.03), maxX]).range([0, width]).nice();
-  const y = scaleLinear().domain([minY, maxY]).range([height, 0]).nice();
-
   const xAxis = axisBottom();
   const yAxis = axisLeft();
 
+  let pad = (maxX - minX) * 0.03;
+  if (minX % 1 === 0 && maxX % 1 === 0) {
```

Comment:
How about we use the JavaScript built-in, [`Number.isInteger()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isInteger) to test if the value is an integer?